### PR TITLE
[Maintenance] skip v1 and other errors

### PR DESCRIPTION
**Pull request by bh2smith**
A bunch of little changes that have been piling up locally for quite some time.

V1 has now over 205 tokens whose symbols and decimals we are fetching on every script run. This stops that.

### CODE REVIEW

File: src/missing_tokens.py [hunk 1]
```diff
@@ -105,34 +105,42 @@ def fetch_missing_tokens(dune: DuneClient, network: Network) -> list[Address]:
...
         )
-        token_details = {}
+        token_details, ignored = {}, set()
         for token in missing_tokens.get_all_tokens():
             try:
                 # TODO batch the eth_calls used to construct the token contracts.
                 token_details[token] = TokenDetails(address=token, w3=w3)
             except web3.exceptions.BadFunctionCallOutput:
-                print(f"Something wrong with token {token} - skipping.")
+                ignored.add(token)
```
Comment (gentrexha) [inline src/missing_tokens.py:126]: Why keep a list of ignored tokens and not just add it to the list instead?

File: src/missing_tokens.py [hunk 2]
```diff
@@ -105,34 +105,42 @@ def fetch_missing_tokens(dune: DuneClient, network: Network) -> list[Address]:
 def run_missing_tokens(chain: Network) -> None:
```
Comment (gentrexha) [inline src/missing_tokens.py:105]: I know this isn't used that often but nitpick suggestion: I'm in favor of removing v1 entirely from here and just have v2 missing tokens data being fetched and have a separate function along the lines of `run_missing_legacy_tokens` or something.

**Review by gentrexha — APPROVED** [review]
Love that we're skipping v1 because I last used it quite a while ago.

Left some discussion comments.

File: src/missing_tokens.py (hunk 1 above)
Comment (bh2smith) [reply src/missing_tokens.py:126]: Not sure I understand your question, but we keep it and then log it at the end of the script (these entries need to be manually added when they occur).

File: src/missing_tokens.py (hunk 2 above)
Comment (bh2smith) [reply src/missing_tokens.py:105]: Yea that would be great. Feel free to make all those changes if you have time!

**Pull request merged**